cmd/opm/registry/serve.go

```diff
@@ -41,7 +49,10 @@ func newRegistryServeCmd() *cobra.Command {
 	}
 
 	rootCmd.Flags().Bool("debug", false, "enable debug logging")
-	rootCmd.Flags().StringP("database", "d", "bundles.db", "relative path to sqlite db")
+	rootCmd.Flags().StringP(databaseFlag, "d", "bundles.db", "relative path to sqlite db")
+	// TODO(joelanford): Is there a default configs folder, or will every index image need to specify the config
+	//   folder in the entrypoint?
+	rootCmd.Flags().StringP(configsFlag, "c", "", "path to declarative index configs")
```

Comment:
I think a much better UX would be that the `registry` command takes a path as a required operand. That path can either be to a single database file, or to a config directory. Having multiple options when there can only be one source for serve is confusing IMO.

Reply:
Updated to use a positional argument, deprecate the `--database` flag, and use file type detection to load the appropriate registry source.